test/ui-test/contentCreatorUiTestNoCreatorTest.ts

```diff
@@ -8,76 +8,111 @@ import { config, expect } from "chai";
-    await editorView.openEditor("Create Devfile");
-    const devfileWebview = await getWebviewByLocator(
-      By.xpath("//vscode-text-field[@id='path-url']"),
+    const devfileWebview = await openCreateWebview(
+      "Ansible: Create a Devfile",
+      "Create Devfile",
     );
+
     const descriptionText = await (
       await devfileWebview.findWebElement(
         By.xpath("//div[@class='description-div']"),
       )
     ).getText();
-    expect(
-      descriptionText,
-      "descriptionText should contain 'Devfiles are yaml files'",
-    ).to.contain("Devfiles are yaml files");
-    const devfileDestination = await devfileWebview.findWebElement(
-      By.xpath("//vscode-text-field[@id='path-url']"),
-    );
-    expect(devfileDestination, "devfileDestination should not be undefined").not
-      .to.be.undefined;
-    await devfileDestination.sendKeys("~");
-    const devfileNameTextField = await devfileWebview.findWebElement(
-      By.xpath("//vscode-text-field[@id='devfile-name']"),
-    );
-    expect(devfileNameTextField, "devfileNameTextField should not be undefined")
-      .not.to.be.undefined;
-    await devfileNameTextField.sendKeys("test");
-    const createDevfileButton = await devfileWebview.findWebElement(
-      By.xpath("//vscode-button[@id='create-button']"),
-    );
-    expect(createDevfileButton, "createDevfileButton should not be undefined")
-      .not.to.be.undefined;
-    expect(
-      await createDevfileButton.isEnabled(),
-      "createDevfileButton button should be enabled now",
-    ).to.be.true;
+    expect(descriptionText).to.contain("Devfiles are yaml files");
+
+    await checkAndInteractWithField(devfileWebview, "path-url", "~");
+    await checkAndInteractWithField(devfileWebview, "devfile-name", "test");
+
+    await clickButtonAndCheckEnabled(devfileWebview, "create-button");
+
     const overwriteDevfileCheckbox = await devfileWebview.findWebElement(
       By.xpath("//vscode-checkbox[@id='overwrite-checkbox']"),
     );
     await overwriteDevfileCheckbox.click();
-    await createDevfileButton.click();
+
+    await clickButtonAndCheckEnabled(devfileWebview, "create-button");
     await sleep(1000);
-    const devfileClearLogsButton = await devfileWebview.findWebElement(
-      By.xpath("//vscode-button[@id='clear-logs-button']"),
-    );
-    expect(
-      devfileClearLogsButton,
-      "devfileClearLogsButton should not be undefined",
-    ).not.to.be.undefined;
-    await devfileClearLogsButton.click();
-    const resetButton = await devfileWebview.findWebElement(
-      By.xpath("//vscode-button[@id='reset-button']"),
+    await clickButtonAndCheckEnabled(devfileWebview, "clear-logs-button");
+    await clickButtonAndCheckEnabled(devfileWebview, "reset-button");
+
+    await checkAndInteractWithField(devfileWebview, "path-url", "~/test");
+    await checkAndInteractWithField(devfileWebview, "devfile-name", "test");
+    await clickButtonAndCheckEnabled(devfileWebview, "create-button");
+
```

Comment:
Before clicking reset, add one more test to uncheck "Overwrite" and recreate the devcontainer (which will fail - expected). This will provide a bit more coverage.

See here for an example: https://github.com/ansible/vscode-ansible/pull/1732/files#diff-87cfb0aafd5395ec414531d5cf42578e92cb4995438776ed01207c68c95c6ad0R80-R84